boards/qemu-coreboot-fbwhiptail-tpm1/qemu-coreboot-fbwhiptail-tpm1.config

```diff
@@ -10,9 +10,12 @@ export CONFIG_LINUX_VERSION=5.10.5
 #export CONFIG_RESTRICTED_BOOT=y
 #export CONFIG_BASIC=y
 
+#Enable HAVE_GPG_KEY_BACKUP to test GPG key backup drive (we cannot inject config under QEMU (no internal flashing))
```

Comment:
:thinking: What if we had a qemu-user-config file somewhere in the source tree that was injected as config.user in qemu builds?  So you could customize the configuration

Reply:
well https://github.com/osresearch/heads/pull/1515#discussion_r1378137635 does it at runtime, which is what we want to test, which should be flashed but is now skipped under qemu boards.

@JonathonHall-Purism can be tested as of [c4a94b3](https://github.com/osresearch/heads/pull/1515/commits/c4a94b34ab9a69efa8298b0fd4e406cfc9831b18) thanks again for the pointers! it was p256 indead and the lack of set_user_config call for that algo. With logic being specific only for algo in checks, everything works as expected now.